### PR TITLE
Convert matrix to string module

### PR DESCRIPTION
**Pull request by mhansen1**
Everything seems to be working ok. Added in a test network that uses the PrintDatatype module. 

Also, added a converters namespace that has the ConvertMatrixToString module/algorithm files. If this isn't wanted/needed I can easily take it out.

### CODE REVIEW

File: src/Modules/Factory/ModuleFactoryImpl1.cc [hunk 1]
```diff
@@ -193,6 +195,7 @@ void ModuleDescriptionLookup::addEssentialModules()
   addModuleDesc<ConvertIndicesToFieldData>("Real ported module", "..."); 
   addModuleDesc<SolveInverseProblemWithTikhonov>("...", "...");
   addModuleDesc<ShowColorMapModule>("ShowColorMap", "Visualization", "SCIRun", "Real ported module", "..."); 
+  addModuleDesc<ConvertMatrixToString>("ConvertMatrixToString","Converters","SCIRun","...","...");
```
Comment (dcwhite) [inline src/Modules/Factory/ModuleFactoryImpl1.cc]: @mhansen1 This is an acceptable factory diff. Although you'll need to merge again, with your own change from the other PR, and with master, so be careful.

File: MatrixToString_test_network/Converter_test.srn5 [hunk 2]
```diff
@@ -0,0 +1,341 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```
Comment (dcwhite) [inline MatrixToString_test_network/Converter_test.srn5]: @mhansen1 Please move this test network to SCIRun/src/ExampleNets/regression

File: src/Core/Algorithms/Factory/HardCodedAlgorithmFactory.cc [hunk 3]
```diff
@@ -95,6 +96,7 @@ using namespace SCIRun::Core::Algorithms::DataIO;
 using namespace SCIRun::Core::Algorithms::Math;
 using namespace SCIRun::Core::Algorithms::BrainStimulator;
 using namespace SCIRun::Core::Algorithms::Math;
+using namespace SCIRun::Core::Algorithms::Converters;
```
Comment (dcwhite) [inline src/Core/Algorithms/Factory/HardCodedAlgorithmFactory.cc]: @mhansen1 You're missing the line to actually connect the module and algorithm. Running your test network results in an error leading to this file: 

ERROR: Null algorithm object, make sure AlgorithmFactory knows about this module's algorithm types.

**Comment by dcwhite** [comment]
Maybe you're missing a commit, assuming your network works...

**Comment by mhansen1** [comment]
Ya, I'm still missing a commit. Trying to get the Laplacian matrix module
to get a good merge so that I can merge the two (making sure that it builds
fine still). You wanted me to merge the Laplacian matrix module with the
ConvertMatrixToString, right?

**Comment by dcwhite** [comment]
Let's keep them separate; if you get this one good, I will merge to master, then you can merge master into the Laplacian branch.

**Pull request merged**